library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -236,25 +237,25 @@ def device_type(dev)
 
     # Detects a subtype of Ethernet device type according /sys or /proc content
     def GetEthTypeFromSysfs(dev)
-      sys_dir_path = Builtins.sformat("/sys/class/net/%1/", dev)
-
-      if FileUtils.Exists(Ops.add(sys_dir_path, "wireless"))
-        return "wlan"
-      elsif FileUtils.Exists(Ops.add(sys_dir_path, "phy80211"))
-        return "wlan"
-      elsif FileUtils.Exists(Ops.add(sys_dir_path, "bridge"))
-        return "br"
-      elsif FileUtils.Exists(Ops.add(sys_dir_path, "bonding"))
-        return "bond"
-      elsif FileUtils.Exists(Ops.add(sys_dir_path, "tun_flags"))
-        return "tap"
-      elsif FileUtils.Exists(Ops.add("/proc/net/vlan/", dev))
-        return "vlan"
-      elsif FileUtils.Exists(Ops.add("/sys/devices/virtual/net/", dev)) &&
-          Builtins.regexpmatch(dev, "dummy.*")
-        return "dummy"
+      sys_dir_path = "/sys/class/net/#{dev}/"
```

Comment:
NP: I'd prefer not to put trailing '/'. It is simply my personal preference to read subsequent usages as ```#{sys_dir_path}/whatever_file```